### PR TITLE
initial commit

### PR DESCRIPTION
**Pull request by serhiicode**
- [DEMO LINK](https://serhiicode.github.io/layout_product-cards/)
- [TEST REPORT LINK](https://serhiicode.github.io/layout_product-cards/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -1,13 +1,46 @@
...
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width,
+            user-scalable=no,
+            initial-scale=1.0,
+            maximum-scale=1.0,
+            minimum-scale=1.0"
+    >
```
Comment (marina-tilniak) [inline src/index.html:11]:
```suggestion
    <meta
      name="viewport"
      content="width=device-width,
        user-scalable=no,
        initial-scale=1.0,
        maximum-scale=1.0,
        minimum-scale=1.0"
    >
```

File: src/index.html [hunk 2]
```diff
@@ -1,13 +1,46 @@
...
+      </div>
+      <div class="card__price">
+        <div class="card__price-text">Цена:</div>
+        <div class="card__price-num">69 999 грн</div>
+      </div>
+      <div class="card__button">
+        <a class="card__buy"
+          href="#"
+          data-qa="hover">
+            Купить
```
Comment (marina-tilniak) [inline src/index.html:41]:
```suggestion
        <a
          class="card__buy"
          href="#"
          data-qa="hover"
        >
            Купить
```

File: src/style.css [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+  height: 16px;
+  margin-right: 4px;
+}
+
+.stars--1 > .stars__star:nth-child(-n+1),
+.stars--2 > .stars__star:nth-child(-n+2),
+.stars--3 > .stars__star:nth-child(-n+3),
+.stars--4 > .stars__star:nth-child(-n+4),
+.stars--5 > .stars__star:nth-child(-n+5) {
+  background-image: url("images/star-active.svg");
```
Comment (marina-tilniak) [inline src/style.css:120]:
```suggestion
.stars--1 :nth-child(-n+1),
.stars--2 :nth-child(-n+2),
.stars--3 :nth-child(-n+3),
.stars--4 :nth-child(-n+4),
.stars--5 :nth-child(-n+5) {
  background-image: url("images/star-active.svg");
```

**Review by marina-tilniak — CHANGES_REQUESTED** [review]

File: src/index.html (hunk 1 above)
Comment (marina-tilniak) [reply src/index.html:11]: ?

**Review by marina-tilniak — APPROVED** [review]

**Review by ArturErmolenko — APPROVED** [review]

**Pull request closed without merging**